core/capabilities/ccip/ccip_integration_tests/integrationhelpers/integration_helpers.go

```diff
@@ -237,11 +238,22 @@ func (t *TestUniverse) AddCapability(p2pIDs [][32]byte) {
 	}
 }
 
-func NewHomeChainReader(t *testing.T, cr types.ContractReader, ccAddress common.Address) ccipreader.HomeChain {
-	hcr := ccipreader.NewHomeChainReader(cr, logger.TestLogger(t), 50*time.Millisecond, types.BoundContract{
-		Address: ccAddress.String(),
-		Name:    consts.ContractNameCCIPConfig,
-	})
+func NewHomeChainReader(
+	t *testing.T,
+	cr types.ContractReader,
+	ccAddress common.Address,
+	chainID string,
```

Comment:
It could be anything that is an identifier (for evm it's going to be integer). ChainSelectors doesn't make sense for monitoring, because they are randomized and very hard to analyze dashboards with chainSelectors instead of chainID

Reply:
We already have `common.Address` here, so no need to make it agnostic at this stage. This is also used only in test code